src/styles/card.scss

```diff
@@ -0,0 +1,118 @@
+@import "../utils/variables.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)